Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -180,9 +180,10 @@ private void OnGameChange(GameRunLevelChangedEvent ev)
         }
 
         bool shouldCapitalize = (desiredType != InGameICChatType.Emote);
+        bool shouldCapitalizeTheWordI = true;
```

Comment:
Could you make this a simple comparison on the current locale set (e.g. "en-*")? That would fix this accidentally breaking things for translated forks.

Reply:
Seems reasonable. I don't know how to do it but I'll figure it out quick.

Reply:
I finished making a culture check but I can hardly call it "simple", I'll leave this unresolved until I can get help with making it simpler. It's difficult to test these things.